GVRf/Framework/framework/src/main/jni/objects/textures/image.h

```diff
@@ -160,6 +160,8 @@ class Image : public HybridObject
     std::mutex mUpdateLock;
     short   mType;
     short   mLevels;
+    int   mXOffset;
```

Comment:
I think explicit initialization to 0 is necessary.

Reply:
OK